Counterexamples/AharoniKorman.lean

```diff
@@ -59,7 +59,7 @@ aim of reaching a contradiction (as then, no such partition can exist). We may f
   them is exactly the length of a maximal chain between them (`card_C_inter_Icc_eq`).
 * For two points of `C` in the same level, and two points `(a, b, n)` and `(c, d, n)` between them,
   if `a + b = c + d` then `f (a, b, n) = f (c, d, n)` (`apply_eq_of_line_eq`).
-* No element of `S ​\ (C ∩ level n)` can be mapped into `level (n + 1)` (`not_S_hits_next`). This
+* No element of `S \ (C ∩ level n)` can be mapped into `level (n + 1)` (`not_S_hits_next`). This
```

Comment:
Somehow a zero-width space ended up in here.

Reply:
Wow, good catch! Did you search all of mathlib for non-printing whitespace chars?

Reply:
Woah, nice catch! I think I know how this happened: I have `\ ` set locally to produce a zero-width space, which means when I write set-difference sometimes, I typo a zero-width space. And in code it usually gives a parse error when I make the typo, but not in comments! Thanks for catching this!